web/html-forms.md

```diff
@@ -0,0 +1,164 @@
+# HTML Forms
+
+### Projected Time
+
+1.5 hours
+
+- Lesson: 30min
+- Guided Practice: 1hour
+- Independent Practice: 2hour
+
+### Prerequisites
+
+- [HTML lesson](./html.md)
+
+### Motivation
+
+- Forms are one of the essential topics in HTML.
+- They are widely used in almost all websites in order to gather the required information of user.
```

Comment:
- They are widely used in almost all websites in order to gather the required information from users.